pkg/sfu/downtrack.go

```diff
@@ -638,7 +643,7 @@ func (d *DownTrack) WriteRTP(extPkt *buffer.ExtPacket, layer int32) error {
 		}
 	}
 
-	d.rtpStats.Update(hdr, len(payload), 0, time.Now().UnixNano())
+	d.rtpStats.Update(hdr, len(payload), 0, extPkt.Arrival)
```

Comment:
Just using the arrival time stamp in the forwarding path so that it hugs the publish side stamping on SFU.